web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -445,7 +445,7 @@ public function work(string $wid): array {
       'publizon-error-status-invalid-phone-number-text' => $this->t('Invalid phone number', [], ['context' => 'Work Page']),
       'publizon-error-status-library-server-not-responding-text' => $this->t("The library's server is not responding – try logging in again later.", [], ['context' => 'Work Page']),
       'publizon-error-status-monthly-loan-limit-reached-text' => $this->t('Your library has reached the limit for the number of loans this month', [], ['context' => 'Work Page']),
-      'publizon-error-status-no-access-because-not-resident-text' => $this->t('You do not have access to eReolen from this library as you are not registered as a resident in the municipality. Contact the library.', [], ['context' => 'Work Page']),
+      'publizon-error-status-no-access-because-not-resident-text' => $this->t('You do not have access to publizon from this library as you are not registered as a resident in the municipality. Contact the library.', [], ['context' => 'Work Page']),
```

Comment:
This is an error exposed to the end user. Most probably the user will not know what Publizon is. Although the string is system english and will be translated you could change the text to something like:
"You do not have access to digital material from this library..."